packages/iov-bcp-types/src/bcp.ts

```diff
@@ -99,6 +127,10 @@ export interface IovReader {
   readonly chainId: () => Promise<ChainId>;
   readonly height: () => Promise<number>;
 
+  // these emits the new blockHeight on every block,
+  // so you can trigger a custom response
+  readonly changeBlock: () => Stream<number>;
```

Comment:
Not for this PR but as I am just seeing this: wouldn't `blockChanges()` be a better name? Method names starting with a verb usually indicate an action. When I implemented Lisk, I didn't have any idea what the three mothods `change*` were supposed to do

Reply:
Nice point. We should make a new pr